AZ3166/src/cores/arduino/UARTClass.cpp

```diff
@@ -20,9 +20,10 @@
 #include "UARTClass.h"
 #include "BufferedSerial.h"
 
-UARTClass::UARTClass()
+UARTClass::UARTClass(const uint8_t p)
 {
   serial = NULL;
+  port = p;
```

Comment:
consider to have a definition for this UART port, default is UART 6, and what you add is UART 1

Reply:
I'm not quite sure what you mean.  I define UART_1 in Peripherals.h are you saying that I should be using that macro here?  Can you elaborate please? :)

Reply:
Yes, my initial thinking is about define a new enumerate.
Use the macro from peripherals.h is also fine.